### ISSUE TITLE
Guided Anchoring RPN

### ISSUE THREAD

**Issue by sehyungp92** [open-mmlab/mmdetection#3043]
Hi,

Many thanks for sharing such a comprehensive toolbox for object detection.

I had a quick question on the Guided-Anchoring RPN. The paper suggests that there were notable gains from discarding the RPN used during training and replacing with pre-computed GA-RPN proposals to finetune (and also used for inference):

- Does this involve training the data using a GA RPN config file then importing the proposals to, for instance, a Mask R-CNN? If so how exactly do you import the proposals to a Mask R-CNN model?
- Do the rest of configs for the Mask R-CNN remain unchanged or do they need to be adjusted (e.g. replace the RPN with a GA RPN)?
- Are there pre-computed proposals for the COCO dataset that can simply be imported?

Kind regards,
Se

**Comment by sehyungp92** [comment]
Any guidance would be much appreciated, thanks.
